### DISCUSSION TITLE
Are all HuggingFace dataset supported in the ScandEval benchmark?

### DISCUSSION THREAD

**Discussion by zhimin-z** [EuroEval/EuroEval#98] [General]
![image](https://github.com/ScandEval/ScandEval/assets/8592144/e3ffd316-092f-414f-880e-4ac1d76d1f96)
Check https://huggingface.co/ScandEval

**Comment by saattrupdan** [comment]
Hi @zhimin-z!

Almost all of those dataset are available in the `scandeval` framework, yes. Note however that the datasets on Hugging Face include datasets that are still undergoing review (see [this PR](https://github.com/ScandEval/ScandEval/pull/95)) or which have been replaced with newer datasets.

To see a list of all the available benchmark datasets, run `scandeval --help` and look at the `--dataset` values. We'll make this more clear in the documentation in the near future 🙂

**Reply by zhimin-z** [reply]
Are these SQUAD datasets `v1` or `v2`?
![image](https://github.com/ScandEval/ScandEval/assets/8592144/47e3a6dd-4c21-4af2-9813-a5631714baf9)

**Reply by saattrupdan** [reply]
> Are these SQUAD datasets `v1` or `v2`?
> 
> ![image](https://github.com/ScandEval/ScandEval/assets/8592144/47e3a6dd-4c21-4af2-9813-a5631714baf9)
> 
> 

They're v2. You can see how all the datasets are built in the scripts directory:

https://github.com/ScandEval/ScandEval/tree/main/src/scripts
